pkg/importer/avro/tests/complex_test.arrai

```diff
@@ -0,0 +1,135 @@
+//test.assert.equal(
```

Comment:
It's hard to recognise that this file is actually a test. Import the fixtures instead, like

```
  //test.assert.equal(//os.file('complex.golden.sysl'), //{./../transformer}("complex.avsc", "Avro", "com.example"))
```